packages/reactotron-app/App/Stores/SessionStore.js

```diff
@@ -144,18 +155,45 @@ class Session {
     return !hidden
   }
 
+  // Sets the selected connection
+  @action
+  setSelectedConnection (connection) {
+    this.selectedConnection = connection
+  }
+
   handleCommand = command => {
     if (command.type === 'clear') {
+      const newCommands = pipe(
```

Comment:
@skellock This might suck... yeah... I think this sucks. I don't think `pipe` is needed to do a single thing like this but when calling reject directly it threw a fit. If you have thoughts on a better way I am allllll ears.

Reply:
That's weird.  Feel free to convert anything basic like this to non-ramda.  `.filter`, `.map`, `.sort`... i think these are all preferable to use the built-in functions rather than to shoe-horn ramda in there.